src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/Table.java

```diff
@@ -388,6 +403,8 @@ private Content toContent() {
                     .put(HtmlAttr.ARIA_ORIENTATION, "horizontal");
 
             HtmlId defaultTabId = HtmlIds.forTab(id, 0);
+            table.put(HtmlAttr.ROLE, "table")
+                 .put(HtmlAttr.ARIA_LABELLEDBY, defaultTabId.name());
```

Comment:
I note that the `aria-labelledby` attribute is already set to the same value on the tab-panel element which contains the table (by code further down in this method). That attribute is also updated dynamically by the code in `script.js` when switching to a different tab. Since the tab-panel is the immediate parent of the table element, and the table its only child, is it actualy necessary to set attribute in both elements?